frontend/amundsen_application/api/metadata/v0.py

```diff
@@ -42,18 +43,31 @@ def _get_table_endpoint() -> str:
     return table_endpoint
 
 
+def _get_feature_endpoint() -> str:
+    feature_endpoint = app.config['METADATASERVICE_BASE'] + FEATURE_ENDPOINT
+    if feature_endpoint is None:
```

Comment:
ouuh I think this was an oversight from the person who wrote the code for other resources too, this will always at least have the /feature, I think the idea was to check for if the metadata service based is defined in the config

Reply:
or maybe this method used to have a parameter for type? I am honestly not sure how we ended up with this behavior looking back at it...